### ISSUE TITLE
Clarify passphrase screen functions at boot via improved text/icons

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#5652]
## Problem
On boot, I am prompted to enter-in two passwords: the disk decryption password, and then my unique user password for the Qubes OS. As a non-technical user, however, I am not likely to understand that the first password is how my hard drive's encryption/decryption is managed. 

## Solution
Swap use of icons and add a few words to both screens. Below is an "ideal" version. A first-step towards it, would be simply adding the word "decryption" to the first screen, while also removing its Qubes icon. Yeah, it'd also just be nice to include the OS version release info bit, in the 2nd screen. The F8 hide/show bit, is a [separate issue here](#5521).

**1st Screen**
![image](https://user-images.githubusercontent.com/8262612/74574979-2c690900-4f3a-11ea-9a28-38516f7b480c.png)

**2nd Screen**
![image](https://user-images.githubusercontent.com/8262612/74575010-430f6000-4f3a-11ea-8054-5d321b08e9ea.png)

## Why this matters
For non-technical high-risk users coming into Qubes from Mac and Windows environments, breaking certain behavioral habits to embrace stronger opsec practices will be essential for success. Clarity in understanding which parts of the crypto glitterpony fart at what times, will help enable compliant behavior in users. Regular folks also rarely read documentation—they "learn" by doing. As such, it's important for the specifics of disk-decryption vs signing into Qubes as a specific user, to be more clearly framed via UI text and semiotics.

Today, the first password is presented to users (on a screen long overdue for a wipe down) as such: 

**First Screen**
![image](https://user-images.githubusercontent.com/8262612/74575098-92559080-4f3a-11ea-84f4-156b453bd407.png)

**Second Screen**
![image](https://user-images.githubusercontent.com/8262612/74575075-81a51a80-4f3a-11ea-8dfa-53563521ea42.png)

From the above presentation, 2 exceptionally subtle points of confusion arise:
- There's a Qubes logo on the first screen, so I'm thinking this is my entry point into Qubes, when in fact that's the next screen.
- The text says "Disk Password." Ok, great! It's a disk password. I'm a non-technical user... I don't know that there's no such thing as a disk password that doesn't involve encryption. I just figure it's one of the two passwords I was given to use when logging onto my Qubes laptop. 

For SecureDrop Journalist users, ensuring they leave their workstations to rest with their drives encrypted, is important. Understanding when decryption happens, is probably one of the most important pieces in the compliance puzzle for them, as that will reinforce the need to break the standard mac/pc user habit of simply closing the laptop lid and putting the laptop away—vs shutting it down, completely. Because only the latter, puts their hard drive in its encrypted state.

## Why now?
Well, because Erik just informed me that's how it all works. And I'd had no idea, before.  I just figured the screensaver made encryption happen. Because I'm a designer. Not a developer. Gotta call out the cryptopony farts, clearly, as they happen. :D

**Comment by deeplow** [comment]
A better solution I think could be just having one passhprase. Started a related discussion on the forum: [Why does Qubes have 2 passphrases? (disk encryption & login)](https://qubes-os.discourse.group/t/why-does-qubes-have-2-passphrases-disk-encryption-login/4312/1).

**Comment by deeplow** [comment]
Not sure if I should open an issue for having only one password by default or if I should keep it here. I'll leave it here because it seems like different solutions to the same problem.

ProtonMail has an interesting approach to this. By default the user has a single passphrase, but they have to option to add another one:

![proton-2pw](https://user-images.githubusercontent.com/47065258/120540801-9e3bbf00-c3d8-11eb-87cc-57d027b3e570.png)

**Comment by ninavizz** [comment]
@deeplow I feel the thing you're speaking to, is a user-choice issue, not a technical one?

This issue it to simply enable a keyboard entry to show/hide the user's entered passphrase on FDE. Even if there is a single password being used, it doesn't matter. A long-assed password needs to be seen to be typed in correctly. 

This has continued to be a problem with our SecureDrop Workstation users, and I'd love to see it get prioritized. Andrew just removed the `4.1` milestone and changed it to TBD, so that has me worried.

Could this go in as a 4.0 update, without having to wait for a stable 4.1 release? Will inquire with M2 when they are available.

**Comment by deeplow** [comment]
> @deeplow I feel the thing you're speaking to, is a user-choice issue, not a technical one?

Yes. Above all it's about setting a default (1 password only) and allowing for user choice (if the user is even aware that they need it).

**Comment by ninavizz** [comment]
Created on the Plymouth project's issues repo:
https://gitlab.freedesktop.org/plymouth/plymouth/-/issues/150

**Comment by ben-grande** [comment]
Issue was completed:

- Disk password: https://openqa.qubes-os.org/tests/165357#step/firstboot/2
- Login: https://openqa.qubes-os.org/tests/165357#step/firstboot/135
